src/cmd/logs.rs

```diff
@@ -0,0 +1,62 @@
+//! The `logs` command.
+
+use args; ///::{self, ToArgs};
+use cmd::CommandCompose;
+use command_runner::CommandRunner;
+#[cfg(test)]
+use command_runner::TestCommandRunner;
+use errors::*;
+use pod::{Pod, PodType};
+use project::Project;
+
+/// We implement `logs` with a trait so we put it in its own module.
+pub trait CommandLogs {
+    /// Display logs for a given service
+    fn logs<CR>(&self,
+               runner: &CR,
+               act_on: &args::ActOn,
+               opts: &args::opts::Logs)
+              -> Result<()>
+        where CR: CommandRunner;
+}
+
+impl CommandLogs for Project {
+    fn logs<CR>(&self,
+              runner: &CR,
+              act_on: &args::ActOn,
+              opts: &args::opts::Logs)
+              -> Result<()>
+        where CR: CommandRunner
+    {
+        let pred = |p: &Pod| p.pod_type() != PodType::Task;
+        self.compose(runner, "logs", act_on, pred, opts)
```

Comment:
This will do something strange if `follow` is set and `act_on.len() != 1`, won't it?

If so, could we please display a helpful error message in this case? You can just `return Err(format!(...).into())` to prepare an error message; we don't need any a new error type for an error that will go away soon anyway.